server/src/ankaios_server.rs

```diff
@@ -435,6 +476,14 @@ mod tests {
         });
         assert_eq!(from_server_command, expected_from_server_command);
 
+        assert!(matches!(
+            comm_middle_ware_receiver.recv().await.unwrap(),
+            FromServer::Response(Response {
+                request_id,
+                response_content: ResponseContent::UpdateStateSuccess(UpdateStateSuccess { added_workloads, deleted_workloads })
+            }) if request_id == REQUEST_ID_A && added_workloads == vec!["workload A"] && deleted_workloads.is_empty()
+        ));
```

Comment:
What about writing this:

```suggestion
        assert_eq!(
            comm_middle_ware_receiver.recv().await.unwrap(),
            FromServer::Response(Response {
                request_id: REQUEST_ID_A.into(),
                response_content: ResponseContent::UpdateStateSuccess(UpdateStateSuccess {
                    added_workloads: vec!["workload B".into()],
                    deleted_workloads: Vec::new(),
                }),
            })
        );
```

This gives better (more readable output) in case of failure. But it is a matter of personal taste of course :-)

Reply:
I have applied the suggestion, but I had change `"workload B"` to `"workload A"` first.

Reply:
Oops. This was my mistake. I was trying if the assertion works (and I forgot to revert the change) :-)